transcom/mymove
# MAIN - B-21687 - Inconsistent Design Multiple Tags

## [B-21687](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-21687)

INT PR: https://github.com/transcom/mymove/pull/14461

## Summary

This is to change how Tags look in the app and how it is displayed in the shipment cards. 

- `Tag` component now has border radius of `2em`.
- `DIVERSION` and `DIVERSION REQUESTED` tags are now in light orange/gold.
- `CANCELLATION REQUESTED` tag is now in same color as `CANCELLED` which is light red.
- PPM status tags such as `WAITING FOR CUSTOMER` or `PACKET READY FOR DOWNLOAD` are now in light green.
- All the tags in the shipment card now have the same font size.
- Updated tags in `EvaluationReportShipmentDisplay` to similarly match other shipment card displays.

### How to test

1. Create a move with a HHG and a PPM.
2. Complete counseling as a SC and submit.
3. Either manually activate each tags shown in the screenshot to verify or go to `src/components/Office/ShipmentDisplay/ShipmentDisplay.jsx` and delete all the conditional checks related to `Tag` component below line # 99. Do the same for other areas of the app and verify the colors and rounded edges match the screenshot.
4. This story focuses on the shipment cards, but since the rounded edges have been added to all the `Tag` - verify nothing looks out of the norm.

## Screenshots
![image](https://github.com/user-attachments/assets/5f08d640-57f8-42b7-b2c1-014fbc66bac6)
![image](https://github.com/user-attachments/assets/0893ccbb-4f64-41be-88e9-8c731b800aaf)
![image](https://github.com/user-attachments/assets/7d84e8c6-4d86-4447-b8c1-92bca1afdc1f)
![image](https://github.com/user-attachments/assets/d0b57925-23bf-4ef9-9b61-d6a25e00fd01)
